app/subscriber/src/components/content-list/Attributes.tsx

```diff
@@ -0,0 +1,64 @@
+import { formatDate } from 'features/utils';
+import moment from 'moment';
+import * as React from 'react';
+import { useLookup } from 'store/hooks';
+import { ContentTypeName, IContentModel, Row, Show } from 'tno-core';
+
+export interface IAttributesProps {
+  /** The content item */
+  item: IContentModel;
+  /** Whether to show the date */
+  showDate?: boolean;
+  /** Whether to show the time */
+  showTime?: boolean;
+  /** The view options */
+  viewOptions?: {
+    section: boolean;
+  };
+  /** Whether to add margin spacing */
+  margin?: boolean;
+}
+export const Attributes: React.FC<IAttributesProps> = ({
+  item,
+  showDate,
+  showTime,
+  viewOptions,
+  margin,
+}) => {
+  const noByLine = ['Talk Radio', 'News Radio', 'Events'];
```

Comment:
This will break as soon as an admin changes the names of the media types.

Reply:
Will rewire with settings, forgot to change this